docs/auth/multi-factor-auth.md

```diff
@@ -36,15 +38,15 @@ const phoneOptions = {
 };
 
 // Sends a text message to the user
-const verificationId = await auth().verifyPhoneNumberForMultiFactor(phoneOptions);
+const verificationId = await auth.verifyPhoneNumberForMultiFactor(phoneOptions);
```

Comment:
This is a large deviation from the firebase-js-sdk APIs for modular unfortunately - I'm guessing we've not mapped the implementation correctly

Should use `verifyPhoneNumber` on a `PhoneAuthProvider` object that we've initialized with a `PhoneInfoOptions` object we've created with the MFA session from the exception we got from the first factor 

> Initialize a PhoneInfoOptions object with the user's phone number and the multi-factor session. These values are contained in the resolver object passed to the auth/multi-factor-auth-required error:

https://firebase.google.com/docs/auth/web/multi-factor#signing_users_in_with_a_second_factor

Reply:
I'll have a PR posted shortly that implements the API correctly as `new PhoneAuthProvider(getAuth()).verifyPhoneNumber(<PhoneInfoOptions>)`

Reply:
Just merged #8499 with the API fixes needed to match firebase-js-sdk here - going to release it in a moment but that should unblock this now that the docs can use the correct APIs

Reply:
TYSM for taking care of the migration Mike! I followed the PR somewhat closely and just updated the docs accordingly.

Reply:
Happy to do it - getting the namespace -> modular conversion handled (and handled well...) is our current top priority internally, and your PR was the motivation I needed to take another big swing at it. We're getting closer to done with all the modules...